code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -875,16 +875,37 @@
 
 /datum/reagent/medicine/mutadone
 	name = "Mutadone"
-	description = "Removes jitteriness and restores genetic defects."
+	description = "Removes jitteriness and temporarly supresses genes from manifesting, it also prevents new mutation from appearing while in the system."
 	color = "#5096C8"
 	taste_description = "acid"
+	overdose_threshold = 35
+	var/list/mut = list()
+
+/datum/reagent/medicine/mutadone/on_mob_metabolize(mob/living/carbon/M)
+	..()
+	var/len = M.dna.mutations.len
+	for(var/i in 1 to len)
+		mut += M.dna.mutations[i].type
 
 /datum/reagent/medicine/mutadone/on_mob_life(mob/living/carbon/M)
+	M.adjustStaminaLoss(0.5 *REM, 0)
```

Comment:
undoc'd?

Reply:
not sure if keep it, i wanted to make it feel like chemotherapy

Reply:
I don't care either way, just doc it.